static/templates/actors/hazard/sheet-new.html

```diff
@@ -80,7 +80,7 @@ <h3>{{localize "PF2E.HazardResetLabel"}}</h3>
             <div class="section-container">
                 <div class="section-header">
                     <h3>Actions</h3>
-                    {{#if owner}}
```

Comment:
If these `owner` checks are elsewhere, they should be replaced with checking `editable`

Reply:
these are the last ones I think unless the editor params count

Reply:
It should just be ensured that nothing is editable when viewed from a locked compendium, on pain of server errors.